storage/innobase/include/log0recv.h

```diff
@@ -143,6 +147,15 @@ struct recv_dblwr_t
   bool restore_first_page(
          ulint space_id, const char *name, os_file_t file);
 
+  /** Restore the first page of the given tablespace from
+  doublewrite buffer.
+  1) Find the page which has page_no as 0
+  2) Read first 3 pages from tablespace file
+  3) Compare the space_ids from the pages with page0 which
+  was retrieved from doublewrite buffer
+  @return space_id or 0 in case of error */
+  uint32_t find_first_page(const char* name, os_file_t file);
```

Comment:
Neither `name` nor `file` are documented with `@param`. Shouldn’t the type be `pfs_os_file_t`? The parameters are in opposite order of those of `init_or_load_pages()`.